### PR TITLE
[BD] Run analytics api in python3

### PR DESCRIPTION
**Pull request by morenol**
## Description
Changes needed to install analytics_api with python3.5

## Reviewers
- [x] @andrey-canon
- [x] Ready for edX review
- [x] @jmbowman

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4278](https://openedx.atlassian.net/browse/OSPR-4278) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: playbooks/roles/analytics_api/meta/main.yml [hunk 1]
```diff
@@ -32,7 +32,6 @@ dependencies:
     edx_django_service_gunicorn_port: '{{ analytics_api_gunicorn_port }}'
     edx_django_service_django_settings_module: '{{ ANALYTICS_API_DJANGO_SETTINGS_MODULE }}'
     edx_django_service_environment_extra: '{{ analytics_api_environment }}'
-    edx_django_service_use_python3: false
```
Comment (andrey-canon) [inline playbooks/roles/analytics_api/meta/main.yml:35]: I think it is True by default, or should we be explicit ?

Comment (morenol) [reply playbooks/roles/analytics_api/meta/main.yml:35]: It is true by default https://github.com/edx/configuration/blob/125ea22bc501b9b5dc231eee3c57436467e8955d/playbooks/roles/edx_django_service/defaults/main.yml#L6

**Review by andrey-canon — APPROVED** [review]

**Review by jmbowman — APPROVED** [review]
@macdiesel Most of the impact of `edx_django_service_use_python3` is in https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/tasks/main.yml .  It installs the two packages being removed here, they were added early for testing.

**Pull request merged**